classes/ActionScheduler_ListTable.php

```diff
@@ -318,10 +318,43 @@ protected function maybe_render_actions( $row, $column_name ) {
 	 * Renders admin notifications
 	 *
 	 * Notifications:
-	 *  1. When the maximum number of tasks are being executed simultaneously
-	 *  2. Notifications when a task is manually executed
+	 *  1. When the maximum number of tasks are being executed simultaneously.
+	 *  2. Notifications when a task is manually executed.
+	 *  3. Tables are missing.
 	 */
 	public function display_admin_notices() {
+		global $wpdb;
+
+		if ( is_a( $this->store, 'ActionScheduler_HybridStore' ) || is_a( $this->store, 'ActionScheduler_DBStore' ) ) {
+			$table_list = array(
+				'actionscheduler_actions' => 'action_id',
+				'actionscheduler_logs' => 'action_id',
+				'actionscheduler_groups' => 'group_id',
+				'actionscheduler_claims' => 'claim_id',
+			);
+
+			$last_error = $wpdb->last_error;
+			foreach ( $table_list as $table_name => $field_name ) {
+				$id = $wpdb->get_var( "SELECT {$field_name} FROM {$wpdb->prefix}{$table_name} LIMIT 1" ); // phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
```

Comment:
Rather than deliberately causing an SQL error, can we use `SHOW TABLES LIKE` instead?

Reply:
@jeffstieler Thanks for the suggestion. I was thinking of performance but since this only occurs in the dashboard on two low use screens, cleaner code is better for maintainability. This is ready for another review.